boe_min.sql

```diff
@@ -0,0 +1,86 @@
+-- For SQLite DBs
+-- conn.executescript(sql_script_string)
+-- minutes DB schema planning
+-- SQLite is loosely typed, meaning you can put almost any data in any column,
+--  regardless of the defined type. (Except for int PKs. I think.)
+;
+
+CREATE TABLE IF NOT EXISTS minutes(
```

Comment:
In a later issue, let's add comments describing in a sentence or two what entity is contained in each of these tables.